pilot/pkg/features/pilot.go

```diff
@@ -38,6 +38,15 @@ var (
 			"Default is 100, not recommended for production use.",
 	).Get()
 
+	// EnableIstioTags controls whether or not to configure Envoy with support for Istio-specific tags
+	// in trace spans. This is a temporary flag for controlling the feature that will be replaced by
+	// Telemetry API (or accepted as an always-on feature).
+	EnableIstioTags = env.RegisterBoolVar(
+		"PILOT_ENABLE_ISTIO_TAGS",
```

Comment:
Nit: `PILOT_ENABLE_ISTIO_TAGS` -> `PILOT_ADD_ISTIO_TRACING_TAGS`

Reply:
Why is pilot even mentioned? ISTIO_ADD_TRACING_TAGS?

Reply:
`PILOT` prefix is convention in this file. We have `PILOT_TRACE_SAMPLING` just above and vars like `PILOT_INBOUND_PROTOCOL_DETECTION_TIMEOUT` and `PILOT_HTTP10`. I'm happy to change, but was going for consistency.